.travis.yml

```diff
@@ -13,4 +13,6 @@ addons:
 services:
   - postgresql
 
-before_install: gem install bundler -v 1.16.1
+before_install:
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
```

Comment:
🤔 Hayfork doesn't really _need_ a particular version of Bundler — doesn't need to it to be pre-2.0 — it's not like it uses Bundler's API.

How are other gems handling Bundler 2.0's release?

(I think many gems are generated with `bundle gem NEW_GEM` and that would create a line like this in the `.gemspec`:)
> ```ruby
> spec.add_development_dependency "bundler", "~> 1.16"
> ```

Do they change `~>` to `>=`? Or just stop pinning a to version of Bundler altogether?

Reply:
Again, thanks for the review!

Travis CI made a [Twitter announcement](https://twitter.com/travisci/status/1080930935095287808?lang=en) that there would be problems after the release of Bundler 2. The fix in this PR is one of the [recommended fixes](https://docs.travis-ci.com/user/languages/ruby/#bundler-20).

I looked through about a dozen gems and noticed that they were handling bundler on Travis in a variety of ways. 

Here are two examples:

Bootsnap ([gemspec](https://github.com/Shopify/bootsnap/blob/master/bootsnap.gemspec#L38)) ([travis.yml](https://github.com/Shopify/bootsnap/blob/master/.travis.yml))
Shoulda-context ([gemspec](https://github.com/thoughtbot/shoulda-context/blob/master/shoulda-context.gemspec#L25)) ([travis.yml](https://github.com/thoughtbot/shoulda-context/blob/master/.travis.yml#L10))

It might be a good idea to stop pinning to a version of Bundler altogether, or just going with this PR which forces Travis to use 1.16.